kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/KcatClient.java

```diff
@@ -71,9 +72,13 @@ public void produceMessages(String topicName, String bootstrap, String message,
 
         LOGGER.atInfo().setMessage("Producing messages in '{}' topic using kcat").addArgument(topicName).log();
         String name = Constants.KAFKA_PRODUCER_CLIENT_LABEL + "-kcat-" + TestUtils.getRandomPodNameSuffix();
+        String jsonOverrides = getInstance().isOpenshift() ? TestUtils.getJsonFileContent("nonJVMClient_overrides.json") : "";
```

Comment:
Personally I think we should re-name `nonJVMClient_overrides.json` to something like `nonJVMClient_openshift.json`